roles/db-copy/tasks/active-copy.yml

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Active-copy | Check pmon process
+  shell: "set -o pipefail; ps -ef | ( grep pmon || true ) | ( grep -i {{ db_name }} || true ) | ( grep -v grep || true ) | wc -l"
```

Comment:
I think we've talked about this pattern before;  Can we avoid matching databases with additional prefixes or suffixes, but still being functional with RAC?

Reply:
it was changed to {{ oracle_sid }}$ and master already got the changes. I will change it here as well to the same since there are other changes in the file